packages/components/src/components/button-pure/button-pure.tsx

```diff
@@ -112,6 +114,11 @@ export class ButtonPure {
   /** Add ARIA attributes. */
   @Prop() public aria?: SelectedAriaAttributes<ButtonPureAriaAttribute>;
 
+  /** The id of a form element the button should be associated with. */
+  @Prop({ reflect: true }) public form?: string;
```

Comment:
Add comment how reflect is necessary since it won't work in react without attribute